packages/forma-36-react-components/src/components/Asset/Asset.tsx

```diff
@@ -25,6 +26,7 @@ export type AssetProps = {
   title: string;
   type?: AssetType;
   className?: string;
+  status?: AssetState;
```

Comment:
I don't think we need an additional prop we can just default to render Asset if no src is present

Reply:
or even better show a missing image icon

Reply:
> I don't think we need an additional prop we can just default to render Asset if no src is present

From yesterday's meeting it sounds like the src is still there, but when you request it the asset may or may not be accessible anymore. This change sets a consistent behaviour in that instance

Reply:
Updated the PR title to be clearer